wagtail_localize/tests/test_edit_translation.py

```diff
@@ -184,19 +184,27 @@ def test_edit_page_translation(self):
         self.assertEqual(
             [(segment['contentPath'], segment['value']) for segment in props['segments'] if segment['type'] == 'synchronised_value'],
             [
+                ('test_richtextfield.http://example.com', 'http://example.com'),
```

Comment:
To future-proof this, we should add some syntax that makes it clear that the ``.`` in ``example.com`` isn't a content path separator. Perhaps we should quote values like this? For example: ``test_richtextfield."http://example.com"``